### PR TITLE
EREGCSC-1015 Resolve dependabot alerts

### PR DESCRIPTION
**Pull request by PhilR8**

### CODE REVIEW

**Comment by PhilR8** [comment]
All `npm` vulnerabilities in `./e2e`, `./serverless`, and `./serverless/static-assets` have been resolved. See screenshots below.  You can also pull down this repository and check for yourself by navigating to the folders listed above and running `npm audit`.

e2e

serverless

serverless/static-assets

However, once you go into `./regulations/static`, things are a bit different.  There are a 20+ vulnerabilities, almost all of which are related to storybook.

Here is few GitHub issue where other people are having very similar issues:

https://github.com/storybookjs/storybook/issues/16556

Which then links to an issue with create-react-app (that has a bunch of duplicate issues listed):

https://github.com/facebook/create-react-app/issues/11641

Suffice to say, it's a current an ongoing issue.  

However, Dan Abramov (of React/Redux/CreateReactApp renown) has a few things to say about the results of `npm audit`:

https://overreacted.io/npm-audit-broken-by-design/
https://github.com/facebook/create-react-app/issues/11174

Namely: these are **build tool** vulnerabilities and are of little consequence:

**But isn't this just ignoring the problem?**
```
No.

Create React App is a build tool. In other words, it doesn't produce a running Node application. It runs at the build time during development, and produces static assets.

However, npm audit is designed for Node apps so it flags issues that can occur when you run actual Node code in production. That is categorically not how Create React App works.
``` 

The resolution proposed by Dan Abramov is to move all build tool packages into `devDependencies` and run:

`npm audit --production`

And that should show zero vulnerabilities.

File: e2e/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "@testing-library/cypress": "^7.0.5",
     "axe-core": "^4.3.0",
-    "cypress": "^6.9.1"
+    "cypress": "^9.1.1"
```
Comment (PhilR8) [inline e2e/package.json:14]: upgraded to latest version of Cypress.  All tests pass (and fail) as expected.

File: regulations/static/package.json [hunk 2]
```diff
@@ -32,16 +32,20 @@
...
+    "@vue/component-compiler": "^4.2.4",
     "babel-loader": "^8.2.2",
     "eslint": "^8.3.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-vue": "^8.1.1",
-    "prettier": "2.2.1"
+    "prettier": "2.2.1",
+    "webpack": "^5.65.0"
```
Comment (PhilR8) [inline regulations/static/package.json:49]: moving Storybook to latest version and migrating to webpack 5

**Review by knollfear — APPROVED** [review]
LGTM!.  Thanks for doing the research into this.

**Pull request merged**